wgengine/netstack/netstack.go

```diff
@@ -373,7 +372,7 @@ func (ns *Impl) updateIPs(nm *netmap.NetworkMap) {
 		pa := tcpip.ProtocolAddress{
 			AddressWithPrefix: ipp,
 		}
-		if ipp.Address.To4() == "" {
+		if ipp.Address.Unspecified() {
```

Comment:
this isn't equivalent to the prior path, the protocol must be v6 if the address is a specified v6 address as well

Reply:
the commit below fixes this case